### PR TITLE
TASK05: SUBTASK01: userspace program with absolute time + SUBTASK02 module with abs and relative time VitaliyZhyrko

### PR DESCRIPTION
**Pull request by vitaliyzhirko**

### CODE REVIEW

File: 05_timers/part01/test_time.c [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+#define SECS_IN_DAY (24 * 60 * 60)
+
+void show_time(clockid_t clock, const char *name)
+{
+    struct timespec ts, ts2;
+    clock_gettime(clock, &ts);
+    clock_getres(clock, &ts2);
+    long days = ts.tv_sec / SECS_IN_DAY;
+    printf("%s:\n\t\t %10ld.%ld\t resolution: %10jd.%09ld\t", name, ts.tv_sec, ts.tv_nsec, ts2.tv_sec, ts2.tv_nsec);
```
Comment (alexposukhov) [inline 05_timers/part01/test_time.c:12]: Please split this line because it is too long. Please keep  MAX_LINE_SIZE = 80 columns.

**Review by alexposukhov — COMMENTED** [review]
Please add a detailed description line to the commit messageTASK05: SUBTASK01: Add task05_01

**Comment by alexposukhov** [comment]
Please fix all checkpatch issues.

File: 05_timers/part01/test_time.c (hunk 1 above)
Comment (vitaliyzhirko) [reply 05_timers/part01/test_time.c:12]: fixed line length and fix detail description in part01, part02

**Review by alexposukhov — APPROVED** [review]

**Pull request merged**